### PR TITLE
fix codeql issues

### PR DESCRIPTION
**Pull request by guotuofeng**
## Checklist before requesting a review
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.
- [ ] Is this PR including examples changes? If yes, please remember to update [example documentation](https://github.com/microsoft/Olive/blob/main/docs/source/examples.md) in a follow-up PR.
- [ ] Add unit tests for this change.

### CODE REVIEW

File: olive/passes/onnx/vitis_ai/quantize.py [hunk 1]
```diff
@@ -167,7 +167,8 @@ def quantize_static(
         # user needs to call pre-process to optimize the model
         if is_ort_version_below_1_16():
             ort_quantize_args["optimize_model"] = False
-        return ort_quantize_static(**ort_quantize_args)
+        ort_quantize_static(**ort_quantize_args)
+        return
```
Comment (trajepl) [inline olive/passes/onnx/vitis_ai/quantize.py:171]: `return None` as what you did in other files?

File: olive/strategy/search_strategy.py [hunk 2]
```diff
@@ -213,6 +213,7 @@ def _create_searcher(self, search_space_names: List[str]) -> SearchAlgorithm:
             searcher = REGISTRY[self._config.search_algorithm](
                 search_spaces_dict, objectives, higher_is_betters, self._config.search_algorithm_config
             )
+            searcher.initialize()
```
Comment (trajepl) [inline olive/strategy/search_strategy.py:216]: This line is the fixing for ``__init__` method calls overridden method`, right?

Comment (guotuofeng) [reply olive/strategy/search_strategy.py:216]: yeah

File: olive/passes/onnx/vitis_ai/quantize.py (hunk 1 above)
Comment (guotuofeng) [reply olive/passes/onnx/vitis_ai/quantize.py:171]: there is no return value for this function.

Comment (trajepl) [reply olive/passes/onnx/vitis_ai/quantize.py:171]: Then why add the `return`? If no this line, will any linter alert be triggred?

**Review by jambayk — APPROVED** [review]

**Pull request merged**